website/content/docs/agent-and-proxy/proxy/caching/persistent-caches/index.mdx

```diff
@@ -4,25 +4,22 @@ page_title: Vault Proxy Persistent Caching
 description: Vault Proxy Caching
 ---
 
-# Vault proxy persistent caching
+# Vault Proxy persistent caching
 
 Vault Proxy can restore tokens and leases from a persistent cache file created
 by a previous Vault Proxy process. The persistent cache is a BoltDB file that
 includes tuples encrypted by a generated encryption key. The encrypted tuples
 include the Vault token used to retrieve secrets, leases for tokens/secrets, and
```

Comment:
clarify this is for dynamic secrets?

Reply:
It should work for static secrets too, but you're right that the text should be updated here. Thanks, let me update it!